proto/connectrpc/conformance/v1alpha1/config.proto

```diff
@@ -89,5 +113,6 @@ enum StreamType {
   STREAM_TYPE_UNARY = 1;
   STREAM_TYPE_CLIENT_STREAM = 2;
   STREAM_TYPE_SERVER_STREAM = 3;
-  STREAM_TYPE_BIDI_STREAM = 4;
+  STREAM_TYPE_HALF_DUPLEX_BIDI_STREAM = 4;
+  STREAM_TYPE_FULL_DUPLEX_BIDI_STREAM = 5;
```

Comment:
Since it's possible for an implementation to choose to support half-duplex but not full-duplex (like if using HTTP/1.1), this seems like a reasonable split to make.